.github/workflows/go.yml

```
@@ -0,0 +1,36 @@
+    - main
+  push:
+    branches:
+    - master
+    - main
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: [ 1.26.x ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)